overrides/cf.d.ts

```diff
@@ -183,12 +183,26 @@ interface RequestInitCfPropertiesImageDraw extends BasicImageTransformations {
   right?: number;
 }
 
+interface RequestInitCfPropertiesImageTrim extends BasicImageTransformations {
```

Comment:
These shouldn't extend the BasicImageTransformations.

Reply:
Sorry, thought I already fixed this... it was the same issue as above.

Fixing...